CONTRIBUTING.md

```diff
@@ -258,3 +258,16 @@ the main branch. The workflow file should be named as
 [![NuGet version badge](https://img.shields.io/nuget/v/{your_package_name})](https://www.nuget.org/packages/{your_package_name})
 [![NuGet download count badge](https://img.shields.io/nuget/dt/{your_package_name})](https://www.nuget.org/packages/{your_package_name})
 ```
+
+### Guidance for components on supporting target frameworks
+
+* Support the [.NET
```

Comment:
looks like the usage of SHOULD and MUST is completely gone now?

Reply:
it was by mistake - fixed.